tsdb/engine/tsm1/engine.go

```diff
@@ -57,13 +57,13 @@ const (
 
 // Engine represents a storage engine with compressed blocks.
 type Engine struct {
-	mu                         sync.RWMutex
-	done                       chan struct{}
-	snapshotterDone            chan struct{}
-	wg                         sync.WaitGroup
-	snapshotterWg              sync.WaitGroup
-	levelCompactionsEnabled    bool
-	snapshotCompactionsEnabled bool
+	mu           sync.RWMutex
+	wg           sync.WaitGroup
+	done         chan struct{}
+	levelWorkers int
```

Comment:
Is this the number of running goroutines?  Can you add a comment about this field?